WORKSPACE

```diff
@@ -1,12 +1,12 @@
 workspace(name = "com_github_bazelbuild_buildtools")
 
+# 0.5.5
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "1e8e662ab93eca94beb6c690b8fd41347835e8ce0f3c4f71708af4b6673dd171",
-    strip_prefix = "rules_go-2e319588571f20fdaaf83058b690abd32f596e89",
+    strip_prefix = "rules_go-e1c4b58c05e4a6ab67392daf28f3d57e4902f581",
     urls = [
-        "http://mirror.bazel.build/github.com/bazelbuild/rules_go/archive/2e319588571f20fdaaf83058b690abd32f596e89.tar.gz",
-        "https://github.com/bazelbuild/rules_go/archive/2e319588571f20fdaaf83058b690abd32f596e89.tar.gz",
+        "http://mirror.bazel.build/github.com/bazelbuild/rules_go/archive/e1c4b58c05e4a6ab67392daf28f3d57e4902f581.tar.gz",
```

Comment:
why upgrade the version, actually?

Reply:
same reason -- issues with rules_go and sha256

https://github.com/bazelbuild/rules_go/commit/551e1f46cbeb470fe9d085e873c3bb83f075e18c

Reply:
I see. I don't know what's the policy on hashes - they sound like a good idea, but /shrug.